lib/OpenQA/Task/Asset/Limit.pm

```diff
@@ -86,13 +86,17 @@ sub _limit {
 
         my $age_in_seconds = ($now->epoch() - DateTime::Format::Pg->parse_datetime($asset->{t_created})->epoch());
         my $asset_name     = $asset->{name};
-        if ($age_in_seconds >= $untracked_assets_storage_duration || !$asset->{size}) {
+
```

Comment:
not a friend of arbitrarily added blank lines…

Reply:
A blank after a block of variable declarations makes sense to me.